static/app/components/structuredEventData/index.tsx

```diff
@@ -380,3 +398,10 @@ const ValueNumber = styled('span')`
 const ValueObjectKey = styled('span')`
   color: var(--prism-keyword);
 `;
+
+const StyledCopyButton = styled(CopyToClipboardButton)`
+  position: absolute;
+  right: ${space(1.5)};
+  z-index: 2;
+  margin: -${space(0.5)} ${space(1)};
```

Comment:
If you're absolutely positioning this you shouldn't need a margin, should be able to do it with `top` and `right`